packages/optimizer/lib/DomTransformer.js

```diff
@@ -124,6 +124,7 @@ const CONFIG_DEFAULT = {
   optimizeAmpBind: true,
   optimizeHeroImages: true,
   separateKeyframes: true,
+  config: 'custom',
```

Comment:
You have to explicitly set the fact that it's a custom config?￼

Reply:
This is actually not needed anymore. I had the idea to make caching depend on the config, but decided to keep it simple for now. Removed.